src/main/java/ua/lviv/iot/parkingServer/model/Vehicle.kt

```diff
@@ -1,24 +1,17 @@
 package ua.lviv.iot.parkingServer.model
 
 import java.time.Duration
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
 import ua.lviv.iot.parkingServer.model.enums.VehicleType
 
+@Document("vehicle")
 data class Vehicle(
-    override var id: Long,
     var number: String,
```

Comment:
are you sure that you need all your properties in all you entites var

Reply:
Fixed ✅